RELEASES.md

```diff
@@ -1,4 +1,4 @@
-# Unreleased
```

Comment:
This should leave a `#Unreleased` in its place right?

Reply:
For `RELEASES.md` files I haven't been doing that. I can migrate this to a `CHANGELOG.md` if you want, after the release.

Reply:
Ohhhh, right. Agreed.